app/models/concerns/gender_collection.rb

```diff
@@ -0,0 +1,9 @@
+module GenderCollection
```

Comment:
Keep things seperated, especially as one or the other could be shifted into the contact model later on.